gensim/models/word2vec.py

```diff
@@ -190,7 +190,11 @@ def train_batch_sg(model, sentences, alpha, work=None, compute_loss=False):
 
                 # now go over all words from the (reduced) window, predicting each one in turn
                 start = max(0, pos - model.window + reduced_window)
-                for pos2, word2 in enumerate(word_vocabs[start:(pos + model.window + 1 - reduced_window)], start):
+                if model.symmetric:
```

Comment:
You also need to make similar changes for `*_inner.pyx` files, because this doesn't affect to "fast" versions